cicd/README.md

```diff
@@ -53,3 +53,7 @@ Finally, all of the above need some Roles to exist in the AWS accounts before we
 
 * "deploy-setup.sh"
 * "setup.template.yml" - AWS resources for the Setup infrastructure
+
+## Opportunities
+
+- Design more granular notifications by calling Slack webhooks _within_ steps.
```

Comment:
Is this straightforward to do? (and should we log a JIRA if so?)

Reply:
I think it's Jira worthy. Straightforward, but iut's just a different mechanism. We would need to implement code, likely in Bash, that posts to the same SNS topic.